src/Features/CSharp/Portable/CodeFixes/AddImport/CSharpAddImportCodeFixProvider.cs

```diff
@@ -22,125 +22,142 @@
 
 namespace Microsoft.CodeAnalysis.CSharp.CodeFixes.AddImport
 {
-    using SymbolReference = ValueTuple<INamespaceOrTypeSymbol, MetadataReference>;
+    using Packaging;
+    using static AddImportDiagnosticIds;
 
-    [ExportCodeFixProvider(LanguageNames.CSharp, Name = PredefinedCodeFixProviderNames.AddUsingOrImport), Shared]
-    internal class CSharpAddImportCodeFixProvider : AbstractAddImportCodeFixProvider<SimpleNameSyntax>
+    internal static class AddImportDiagnosticIds
     {
         /// <summary>
         /// name does not exist in context
         /// </summary>
-        private const string CS0103 = "CS0103";
+        public const string CS0103 = "CS0103";
```

Comment:
It appears to use the `FixableTypeIds` but not the actual string constants themselves